Workade/Views&ViewModels/Map/MapViewController.swift

```diff
@@ -98,12 +98,13 @@ class MapViewController: UIViewController {
             topInfoStackView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 44)
         ])
         
-        view.addSubview(naverMapButton)
+        view.addSubview(markerInfoView)
+        markerInfoView.isHidden = true
         NSLayoutConstraint.activate([
-            naverMapButton.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -80),
-            naverMapButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
-            naverMapButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 10),
-            naverMapButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -10)
+            markerInfoView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -96),
```

Comment:
[P2] 이런 경우 topAnchor도 bottom을 기준으로 잡기보단, 높이가 80이란 것은 고정값인 것 같으니, 아래처럼 height로 주는 것이 가독성이 더 좋아보입니다.
```suggestion
            markerInfoView.heightAnchor.constraint(equalToConstant: 80),
```

Reply:
수정했습니다!